readthedocs/rtd_tests/tests/test_project_forms.py

```diff
@@ -598,3 +599,33 @@ def test_create(self):
         self.assertEqual(EnvironmentVariable.objects.count(), 2)
         self.assertEqual(EnvironmentVariable.objects.first().name, 'ESCAPED')
         self.assertEqual(EnvironmentVariable.objects.first().value, r"'string escaped here: #$\1[]{}\|'")
+
+
+class TestRandomProjectsInputForm(TestCase):
+
+    def setUp(self):
+        self.project = get(Project)
+
+    def test_get_max_value(self):
+        self.assertEqual(RandomProjectsInputForm().get_max_value(), 2)
```

Comment:
I don't understand that why this value is coming as **2**. It should be **1**.
Also the result of
`Project.objects.all().count()` here is also **2**.